app/actions/actions.js

```diff
@@ -0,0 +1,26 @@
+import * as types from './types';
+import getWeatherAjax from '../API/weatherAPI';
+
+
+export function changeCity(city) {
+    return {
+        type: types.SET_CITY,
+        city,
```

Comment:
one of the good solution use  default syntax for actions: return{ type: type, payload: ....}